Frameworks/Foundation/NSHTTPURLResponse.mm

```diff
@@ -85,7 +85,7 @@ - (id)initWithURL:(NSURL*)url statusCode:(NSInteger)statusCode HTTPVersion:(NSSt
 
         // Parse the filename from the Content-Disposition header field.
         NSString* contentDisposition = [allHeaderFields objectForKey:@"Content-Disposition"];
-        NSRange filenameTagPosition = [contentDisposition rangeOfString:@"filename=" options:NSBackwardsSearch];
```

Comment:
The reference platform appears to pull the first filename, not the last one, in case there is more than one.